models/aeif_cond_beta_multisynapse.h

```diff
@@ -51,20 +51,66 @@
+ integrate-and-fire neuron model. It allows an arbitrary number of synaptic
+ rise time and decay time constants. Synaptic conductance is modeled by a
  beta function, as described by A. Roth and M.C.W. van Rossum
  in Computational Modeling Methods for Neuroscientists, MIT Press 2013,
  Chapter 6
 
- The time constants are supplied by two arrays taus_rise and taus_decay for
- the synaptic rise time and decay time, respectively.  Port numbers
- are then automatically assigned and there range is from 1 to n.  (n
- being the index of the last element of the tau_ex and tau_in
- arrays).
+ The number of receptor ports is supplied by the function "n_receptors".
+ The time constants are supplied by two arrays, "taus_rise" and "taus_decay" for
+ the synaptic rise time and decay time, respectively. The synaptic
+ reversal potentials are supplied by the array "E_rev". The port numbers
+ are automatically assigned in the range from 1 to n_receptors.
  During connection, the ports are selected with the property "receptor_type".
 
+ The membrane potential is given by the following differential equation:
+ C dV/dt = -g_L(V-E_L) + g_L*Delta_T*exp((V-V_T)/Delta_T) + I_syn_tot(V, t)
+           - w + I_e
+
+ where:
+ I_syn_tot(V,t) = \sum_i g_i(t) (V - E_{rev,i})
+
+ and the synapse i is excitatory or inhibitory depending on the value of
+ E_{rev,i}.
+
+
+Parameters:
+The following parameters can be set in the status dictionary.
+
+Dynamic state variables:
+  V_m        double - Membrane potential in mV
+  w          double - Spike-adaptation current in pA.
+
+Membrane Parameters:
+  C_m        double - Capacity of the membrane in pF
+  t_ref      double - Duration of refractory period in ms.
+  V_reset    double - Reset value for V_m after a spike. In mV.
+  E_L        double - Leak reversal potential in mV.
+  g_L        double - Leak conductance in nS.
+  I_e        double - Constant external input current in pA.
+
+Spike adaptation parameters:
+  a          double - Subthreshold adaptation in nS.
+  b          double - Spike-triggered adaptation in pA.
+  Delta_T    double - Slope factor in mV
+  tau_w      double - Adaptation time constant in ms
+  V_th       double - Spike initiation threshold in mV
+  V_peak     double - Spike detection threshold in mV.
+
+Synaptic parameters
+  E_rev      double vector - Reversal potential in mV.
+  taus_rise  double vector - Rise time of synaptic conductance in ms (beta
+                      function).
+  taus_decay double vector - Decay time of synaptic conductance in ms (beta
+                      function).
+
+Integration parameters
+  gsl_error_tol  double - This parameter controls the admissible error of the
+                          GSL integrator. Reduce it if NEST complains about
+                          numerical instabilities.
+
  Examples:
  % PyNEST example, of how to assign synaptic rise time and decay time
  % to a receptor type.
```

Comment:
I think the comment here is not needed. If you want to keep it, mark the comment with Python's `#` instead of `%`, which introduces comments in SLI.